test/libsolidity/syntaxTests/viewPureChecker/eof/inline_assembly_instructions_disallowed_pure_byzantium.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+    function f() public pure {
+        assembly {
+            pop(extstaticcall(0, 1, 2))
+        }
+    }
+}
+// ====
+// EVMVersion: >=byzantium
+// bytecodeFormat: >=EOFv1
```

Comment:
You can merge this case with `inline_assembly_instructions_disallowed_pure.sol`, because EOF is not available on byzantium anyway so the restriction is unnecessary.

Reply:
done